metrotranscom/doorway
# refactor: remove unused doorway uic components

This PR addresses https://github.com/bloom-housing/Bloom-Doorway/issues/27


Related to the dependency issues

## Description

Removes unused `doorway-ui-components`. I checked out `Find all references` for each component. The vast majority weren't used anywhere, so I deleted the file. Another good portion were used, but the files were identical to UIC, so I just pulled in the UIC version and deleted the file. For any components that were used that had Doorway customizations, even if they were really small, I just left them in to tackle later and kept the file.

As a result of some of the CSS being removed, there were a handful of places I needed to update classes in the public site, but it was just around shifting semibold --> bold to maintain the current design. The other change was an a11y fix that was flagged by a linter.

## How Can This Be Tested/Reviewed?

The only place there were slight changes was in the components that we use on all the help / information pages under `Help Center` and `Partners`, so I would check those out and compare the styles to prod.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
